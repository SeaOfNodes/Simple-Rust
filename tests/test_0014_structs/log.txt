directory: tests/test_0014_structs

command: ro "main.ro"
status: 0

command: main
status: None
