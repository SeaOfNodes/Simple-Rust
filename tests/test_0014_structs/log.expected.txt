directory: tests/test_0014_structs

command: ro "main.ro"
status: 0

command: main
status: 5
stdout: initial
stdout: ok
stdout: initial
stdout: ok
stdout: initial
stdout: ok
stdout: modified
stdout: ok
stdout: 
